drivers/i2c/i2c_ll_stm32_device.h.jinja2

```diff
@@ -0,0 +1,8 @@
+{% import 'i2c_ll_stm32.meta.jinja2' as i2c_stm32 %}
+
+#ifndef _I2C_LL_STM32_DEVICE_H_
```

Comment:
Why do we need the ifdef protection, this file should only ever be included once?

Reply:
Hehe, that more or less happens by it self when I start to write something called .h :) But for sure, we do not need it.